pkg/harvester-manager/machine-config/harvester.vue

```diff
@@ -712,6 +715,22 @@ export default {
       }
     },
 
+    async getEnabledVGpuDevices() {
+      const clusterId = get(this.credential, 'decodedData.clusterId');
+      const url = `/k8s/clusters/${ clusterId }/v1`;
+
+      if (url) {
```

Comment:
Should we test for `clusterId` instead? I don't see any situation where `url` would be falsy given the statements above.

Reply:
That's true, thanks!